src/components/IssueReport.jsx

```diff
@@ -0,0 +1,77 @@
+import platform from 'platform'
+import React, { Component } from 'react'
+import FontAwesome from 'react-fontawesome'
+import { Input, Textarea, Box, Button, Modal } from 'resin-components'
+import { submitPensieveIssue } from '../services/github'
+
+console.log(platform)
+
+export default class IssueReport extends Component {
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      showModal: false,
+      loading: false,
+      title: '',
+      body: ''
+    }
+  }
+
+  submit () {
+    this.setState({ loading: true })
+    const body = `
+${this.state.body}
+
+*Sent from: ${platform.description}*
```

Comment:
There's part of me that thinks it'd be "nicer" to default this to the first line of the issue body when first creating the issue (IYSWIM) rather than 'sneakily' appending it to the issue body when you click on the submit button? But perhaps that's just me ;-)

Also, if the user could see that their browser is already auto-detected, then conscientious issue-reporters wouldn't need to manually add that info to the issue body? \*shrug\*